docs/release-notes-8.0.md

```diff
@@ -147,12 +223,234 @@
   - `qx.ui.basic.Atom`, property `show`, renamed to `showFeatures`
   - `qx.ui.mobile.basic.Atom`, property `show`, renamed to `showFeatures`
 
+- Classes that extend `qx.core.Object` (or any subclass) MUST call `super()` in their constructor before accessing properties or setting property values. If `super()` is not called, you will see warnings like `"No $$propertyValues on [ClassName]: possibly missing call to super() in the constructor"`. Make sure all your constructors include a `super()` call at the beginning.
+
+  Example:
+  ```javascript
+  construct() {
+    super();  // Required in v8!
+    this.setMyProperty("value");
+  }
+  ```
+
+- Any property setters called BEFORE `super()` or `this.base(arguments)` will be reset when the parent constructor executes. You must move all property setter calls to occur after the parent constructor invocation.
+
+  v7 (worked but now broken in v8):
+  ```javascript
+  construct(svg, width, height) {
+    this.setWidth(width);   // These values will be lost!
+    this.setHeight(height);
+    this.base(arguments);
+  }
+  ```
+
+  v8 (correct approach):
+  ```javascript
+  construct(svg, width, height) {
+    this.base(arguments);   // Call parent first!
+    this.setWidth(width);   // Now set properties
+    this.setHeight(height);
+  }
+  ```
+
 - Because the entire class and property system was rewritten, there
   may be other obscure backward-compatibility changes that pop up.
   Those listed above are the only ones that reared their heads while
   confirming that the entire qooxdoo test suite successfully runs to
   completion.
```

Comment:
I didn't see any mention of the fact that property names and members are now in the same namespace, so any classes which defined both a property with name `x` and a member variable with name `x` will need to change the name of one of them. (Does `qx migrate` identify and/or modify code to correct for this?)